ClientCore/ClientConfiguration.cs

```diff
@@ -278,23 +278,26 @@ private List<TranslationGameFile> ParseTranslationGameFiles()
         {
             List<TranslationGameFile> gameFiles = new();
 
-            for (int i = 0; clientDefinitionsIni.KeyExists(TRANSLATIONS, $"GameFile{i}"); i++)
+            foreach (string key in clientDefinitionsIni.GetSectionKeys(TRANSLATIONS))
             {
-                // the syntax is GameFileX=path/to/source.file,path/to/destination.file[,checked]
-                string value = clientDefinitionsIni.GetStringValue(TRANSLATIONS, $"GameFile{i}", string.Empty);
-                string[] parts = value.Split(',', StringSplitOptions.RemoveEmptyEntries).Select(s => s.Trim()).ToArray();
+                // syntax must bet GameFileX=..., where X - any text.
```

Comment:
```suggestion
                // the syntax is GameFileX=path/to/source.file,path/to/destination.file[,checked]
                // where X can be any text or number
```
Please don't erase info without real need.